guidelines/enhancement_template.md

```diff
@@ -19,6 +19,20 @@ superseded-by:
   - "/enhancements/our-past-effort.md"
 ---
 
+Start by filling out this header template with metadata for this enhancement.
+
+* `reviewers`: This can be anyone that has an interest in this work.
+
+* `approvers`: All enhancements must be approved, but the appropriate people to
+  approve a given enhancement depends on its scope.  If an enhancement is
+  limited in scope to a given team or component, then a peer or lead on that
+  team is an appropriate approver.  If an enhancement captures something more
+  broad in scope, then a member of the OpenShift architects team or someone
+  they delegate would be appropriate.  Examples would be something that changes
+  the definition of OpenShift in some way, adds a new required dependency, or
+  changes the way we support our customers.  Use your best judgement to
+  determine the level of approval needed and ask for help if you’re not sure.
```

Comment:
Do we want to recommend an "ask for help" channel?  Or how about "Leave it blank if you're not sure, and reviewers will give you suggestions."?

Reply:
I like leaving it blank and asking for input if you're not sure.